packages/vite-plugin-cloudflare/src/node-js-compat.ts

```diff
@@ -43,21 +41,38 @@ export function isNodeCompat(
 }
 
 /**
- * If the current environment needs Node.js compatibility,
- * then inject the necessary global polyfills into the code.
+ * Get a set of module specifiers for all possible Node.js compat polyfill entry-points
+ */
+export function getNodeCompatEntries() {
+	const entries = new Set<string>(Object.values(env.alias));
+	for (const globInject of Object.values(env.inject)) {
+		if (typeof globInject === "string") {
+			entries.add(globInject);
+		} else {
+			entries.add(globInject[0] as string);
+		}
+	}
```

Comment:
```suggestion
	for (const globalInject of Object.values(env.inject)) {
		if (typeof globalInject === "string") {
			entries.add(globalInject);
		} else {
			assert(globalInject[0]);
			entries.add(globalInject[0]);
		}
	}
```

Reply:
Sorry to be picky but I would rather `globInject` is renamed to `globalInject`. `globInject` just makes me think it's something to do with globs.

Reply:
2c335f21ddba84e69051d9a74a5a1a9c6be997df